bots/npm-update

```diff
@@ -85,7 +85,7 @@ def run(package, verbose=False, **kwargs):
     if package:
         packages = [ package ]
     else:
-        packages = data["dependencies"].keys()
+        packages = list(data["dependencies"].keys())
```

Comment:
`list()` it's not necessary in almost every place except for when you return an iterator and the callee isn't expecting it, or in this case  where `random.shuffle()` works in-place and thus cannot deal with an iterator.

I. e. this here is the only case where you need `list()`, it can be reverted everywhere else in this PR. (2to3 is way too overzealous here).

Reply:
Fixed.,